dietpi/dietpi-software

```diff
@@ -10811,6 +10852,73 @@ location = /.well-known/caldav  { return 301 /baikal/html/dav.php; }' > /etc/ngi
 
 		fi
 
+		software_id=125 # Synapse Server
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Configuration
+
+			G_EXEC mkdir -p /mnt/dietpi_userdata/synapse
+
+			G_EXEC sudo -u postgres psql -c "create role synapse with login password '$GLOBAL_PW';"
```

Comment:
Let's use a custom password here, a random one like we do in case of Nextcloud/ownCloud. As this is only for automated access from Synapse, there is no need to use the global software password.